### DISCUSSION TITLE
ACAP development documentation?

### DISCUSSION THREAD

**Discussion by FantasticFiasco** [AxisCommunications/discussions#1] [ACAP] [answered]
I'm looking for documentation regarding ACAP development. I've found the different SDKs and their respective repositories with examples, but is there any other documentation I can read?

**Comment by pataxis** [comment, marked as answer]
There are currently two versions of ACAP SDK, version 3 and 4, where the latter is actively developed.

## ACAP version 3 SDK

- Documentation is found on https://help.axis.com/acap-3-developer-guide.

## ACAP version 4 SDK

- Documentation is found on https://axiscommunications.github.io/acap-documentation.
  -  A special section of the documentation address the [differences between ACAP version 3 and 4](https://axiscommunications.github.io/acap-documentation/docs/introduction/acap4-vs-acap3.html).
